extensions/2.0/Khronos/KHR_displaymapping_pq/README.md

```diff
@@ -96,7 +96,7 @@ The following steps shall be performed before outputing the calculated pixel val
 This would normally be done in the fragment shader.  
 
 3: Scene aperture  
-If `sceneAperture` is defined, then the scene linear light (normally fragment shader output value) pixel value shall be adjusted as defined:    
+If `aperture` is defined, then the scene linear light (normally fragment shader output value) pixel value shall be adjusted as defined:    
```

Comment:
I will be annoying here but the new name `aperture` is even more confusing as it does not related to what's normally designated by the term `aperture` when it comes to rendering/video/photography. Please consider `exposure` or `lightScale` or anything else.

Reply:
See above comment